extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Pinterest Block.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Subscriptions;
+
+use Jetpack_Gutenberg;
+
+const FEATURE_NAME = 'subscriptions';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers the block for use in Gutenberg
+ * This is done via an action so that we can disable
+ * registration if we need to.
+ */
+function register_block() {
+	jetpack_register_block( BLOCK_NAME );
+
+	Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
```

Comment:
There's another more specific function just for styles that we can use to avoid loading `view.js` since it's practically empty and needed only for building the CSS:

https://github.com/Automattic/jetpack/blob/802497c2806565f209c7aa85efcdadb49e0f905a/class.jetpack-gutenberg.php#L544-L545